app.js

```diff
@@ -98,11 +97,10 @@ app.post('/api/v1/projects', async (request, response) => {
 
 app.patch('/api/v1/palettes/:id', async (request, response) => {
   const { id } = request.params;
-  const { name } = request.body;
+  const name = request.body;
   try {
-    const pallete = await database('palettes').find((pallete) => pallete.id == id);
-    pallete.name = name;
-    !pallete.name ? response.sendStatus(404) : response.status(200).json(pallete);
+    const palleteId = await database('palettes').where('id', id).update(name, 'id');
+    !palleteId ? response.sendStatus(404) : response.status(200).json(palleteId);
```

Comment:
Hey!  I checked, and it looks like you're right about the 200 response, so I'll be sure to change mine!